mbedtls-sys/build/config.rs

```diff
@@ -257,79 +236,78 @@ const DEFAULT_DEFINES: &'static [CDefine] = &[
     ("MBEDTLS_CCM_C",                                     Defined),
-    ("MBEDTLS_CERTS_C",                                   Defined),
     ("MBEDTLS_CHACHA20_C",                                Defined),
     ("MBEDTLS_CHACHAPOLY_C",                              Defined),
     ("MBEDTLS_CIPHER_C",                                  Defined),
     ("MBEDTLS_CMAC_C",                                    Defined),
     ("MBEDTLS_CTR_DRBG_C",                                Defined),
-    ("MBEDTLS_DEBUG_C",                                   Undefined),
+    ("MBEDTLS_DEBUG_C",                                   Undefined), // default: Defined
     ("MBEDTLS_DES_C",                                     Defined),
     ("MBEDTLS_DHM_C",                                     Defined),
     ("MBEDTLS_ECDH_C",                                    Defined),
     ("MBEDTLS_ECDSA_C",                                   Defined),
     ("MBEDTLS_ECJPAKE_C",                                 Defined),
     ("MBEDTLS_ECP_C",                                     Defined),
-    ("MBEDTLS_ENTROPY_C",                                 Undefined),
+    ("MBEDTLS_ENTROPY_C",                                 Undefined), // default: Defined
     ("MBEDTLS_ERROR_C",                                   Defined),
     ("MBEDTLS_GCM_C",                                     Defined),
-    ("MBEDTLS_HAVEGE_C",                                  Undefined),
     ("MBEDTLS_HKDF_C",                                    Defined),
     ("MBEDTLS_HMAC_DRBG_C",                               Defined),
+    ("MBEDTLS_LMS_C",                                     Undefined), // default: Defined
+    ("MBEDTLS_LMS_PRIVATE",                               Undefined),
     ("MBEDTLS_NIST_KW_C",                                 Defined),
     ("MBEDTLS_MD_C",                                      Defined),
-    ("MBEDTLS_MD2_C",                                     Defined),
-    ("MBEDTLS_MD4_C",                                     Defined),
     ("MBEDTLS_MD5_C",                                     Defined),
     ("MBEDTLS_MEMORY_BUFFER_ALLOC_C",                     Undefined),
-    ("MBEDTLS_NET_C",                                     Undefined),
+    ("MBEDTLS_NET_C",                                     Undefined), // default: Defined
     ("MBEDTLS_OID_C",                                     Defined),
-    ("MBEDTLS_PADLOCK_C",                                 Undefined),
+    ("MBEDTLS_PADLOCK_C",                                 Undefined), // default: Defined
     ("MBEDTLS_PEM_PARSE_C",                               Defined),
     ("MBEDTLS_PEM_WRITE_C",                               Defined),
     ("MBEDTLS_PK_C",                                      Defined),
     ("MBEDTLS_PK_PARSE_C",                                Defined),
     ("MBEDTLS_PK_WRITE_C",                                Defined),
     ("MBEDTLS_PKCS5_C",                                   Defined),
```

Comment:
turned it off

Reply:
Just found `pkcs5_pbkdf2_hmac` is only supported by pkcs5. And this function is actively used by some projects now

Reply:
That's rather unfortunate, because we've seen that including PKCS5 enables very questionable code outside the PKCS5 module. We could copy the relevant C code into `mbedtls` directly?